parachain/pallets/dot-app/src/benchmarking.rs

```diff
@@ -65,24 +64,27 @@ benchmarks! {
     // Benchmark `unlock` extrinsic under worst case conditions:
     // * The amount is successfully unlocked
 	unlock {
-        let caller: H160 = ETH_CONTRACT_ADDRESS.into();
-        Address::put(caller);
+        Address::set(Default::default());
```

Comment:
Had to set this to `Default::default::<H160>()` in order to match up with `artemis_dispatch::EnsureEthereumAccount::successful_origin`.